internal/docstore/driver/driver.go

```diff
@@ -71,3 +75,28 @@ type Mod struct {
 	FieldPath []string
 	Value     interface{}
 }
+
+// A Query defines a query operation to find documents within a collection based
+// on a set of requirements.
+type Query struct {
+	FieldPaths []string         // the selected field paths
+	Filters    []Filter         // a list of filter for the query
+	Limit      int              // the number of result in one query request
+	StartAt    interface{}      // define the starting point of the query
```

Comment:
I don't see where a caller would get this value, or what it represents?

Reply:
We can remove it for now, and figure out how to start a query at a particular place.